cmd/logs/main.go

```diff
@@ -55,7 +55,14 @@ func main() {
 		WithGrpcAddress(cfg.GrpcAddress)
 
 	// TODO - add adapters here
-	svc.AddAdapter(adapter.NewDummyAdapter())
+	minioAdapter, err := adapter.NewMinioAdapter(cfg.MinioEndpoint, cfg.MinioAccessKey, cfg.MinioSecretKey, cfg.MinioRegion, cfg.MinioToken, cfg.MinioBucket)
+	if err != nil {
+		log.Errorw("error creating minio adapter", "error", err)
+		svc.AddAdapter(adapter.NewDummyAdapter())
```

Comment:
do we need DummyAdapter in prod?

Reply:
sounds like, log service will not store logs without minio?

Reply:
Yeah I would rename it to LogAdapter or sth and attach it onlly when debug mode is set

Reply:
Good point - it should only try to set minio when it's not in agent mode

Reply:
I wold put whole logic into if when in OSS mode
additionally add additional dummyadapter when DEBUG is set 
and add cloud adapter when Agent key is set

Reply:
got it, I will make the changes then